DependencyInjection/Configuration.php

```diff
@@ -201,6 +201,7 @@ private function configureDbalDriverNode(ArrayNodeDefinition $node) : void
                 ->scalarNode('application_name')->end()
                 ->scalarNode('charset')->end()
                 ->scalarNode('path')->end()
+                ->booleanNode('read_only')->end()
```

Comment:
should define info explaining this, similarly as is done for `persistent` option